publish/publish.proj

```diff
@@ -132,6 +132,24 @@
     <Exec Condition="'@(_DownloadedSymbolsPackages)' != ''" Command="$(NuGetPushSymbolsCommand) %(_DownloadedSymbolsPackages.Identity)" />
   </Target>
   
+  <Target Name="PublishDebToolPackagesToFeed"
+          DependsOnTargets="CheckIfAllBuildsHavePublished"
```

Comment:
You'll want to update the pipeline.json to pass `PublishDebToolToFeed` for the Ubuntu 14.04 leg